src/CSET/operators/constraints.py

```diff
@@ -181,6 +181,39 @@ def generate_time_constraint(
     return time_constraint
 
 
+def generate_area_constraint(
+    lat_start: float, lat_end: float, lon_start: float, lon_end: float, **kwargs
+) -> iris.Constraint:
+    """Generate an area constraint between latitude/longitude limits.
+
+    Operator that takes a set of latitude and longitude limits and returns a
+    constraint that selects grid values only inside that area.
+
+    Arguments
+    ---------
+    lat_start: float
+        Latitude value for lower bound
+    lat_end: float
+        Latitude value for top bound
+    lon_start: float
+        Longitude value for left bound
+    lon_end: float
+        Longitude value for right bound
+
+    Returns
+    -------
+    area_constraint: iris.Constraint
+    """
+    lat_constraint = iris.Constraint(
+        coord_values={"grid_latitude": lambda cell: lat_start < cell < lat_end}
+    )
+    lon_constraint = iris.Constraint(
+        coord_values={"grid_longitude": lambda cell: lon_start < cell < lon_end}
+    )
+    area_constraint = lat_constraint & lon_constraint
+    return area_constraint
```

Comment:
```suggestion
    area_constraint = iris.Constraint(
        coord_values={
            "grid_latitude": lambda cell: lat_start < cell < lat_end,
            "grid_longitude": lambda cell: lon_start < cell < lon_end
            }
    )
    return area_constraint
```
We can do this with a single constraint, as coord_values on a [Constraint](https://scitools-iris.readthedocs.io/en/stable/generated/api/iris.html#iris.Constraint) is a dictionary of all the coordinates. (Let's see if ruff likes my formatting...)